less/index.less

```diff
@@ -4,56 +4,287 @@
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
 	margin: 0;
 	padding: 0;
 	border: 0;
-	font-size: 100%;
-	font: inherit;
+	// font-size: 100%;
+	// font: inherit;
 	vertical-align: baseline;
 }
+
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
 	display: block;
 }
-body {
-	line-height: 1;
+// variables:
+@background-color:#F5F5F5;
+@text-color:#4d004d;
+
+@link-text-color: black;
+@link-background-color:white;
+
+// mixins:
+
+.font-classA{
+	color:white;
+	font-size: 4 rem;
+	text-align:center;
+}
+
+html {
+	font-size: 62.5%;
+	box-sizing: border-box;
+
+}
+
+// body {
+// 	line-height: 1;
+
+// }
+
+.container {
+	width: 90%;
+	margin: 0 auto;
+}
+
+header {
+		display:flex;
+		flex-direction: row-reverse;
+		align-items: end;
+		justify-content: right;
+		
+		nav {
+			margin:20px 0 10px 100px;
+			
+			display: flex;
+			width: 30%;
+			height: auto;
+			justify-content: space-between;
+			text-emphasis: bold;
+
+
+			a { 
+				text-decoration: none;
+				color: @link-text-color;
+				background: @link-background-color;
+				font-size: 2 rem;
+			}
+	
+	}
+}
+
+.main-content {
+	
+	width: 100%;
+	height: 60%;
+	background: @background-color;
+
+		.nameBox{
+		padding:0 30px 30px 30px;
+		font-size: 3 rem;
+		font-family: Verdana, Geneva, Tahoma, sans-serif;
+		}
+		
+
+		.treepic {
+		margin: 30px;
+		height: inherit;
+		width: auto;
+		max-width: 20%;
+
+			img {
+			height: inherit;
+			// width:inherit;
+		}
+	}	
+}
+.middle{ 
+	
+	display:flex;
+	
+	.statement{
+	  
+	  width:50%;
+	  margin: 20px 0 0 20px;
+	//   padding:1%;
+	}
+	.stuff{
+		
+		width:50%;
+		margin: 20px 0 0 20px;
+		li{
+			border:0.5px solid grey;
+			padding:8px;
+		}
+	}
+}
+
+.bottomSection{
+	margin:15px 0 0 16px;
+	table{
+		
+		width:100%;
+			tr{
+				text-align:left;
+				height: 20px;
+				font-weight: bold;
+				color:black;
+				td{
+					// .odd{
+					// 	background-color: white;
+					// }
+					.even{
+						background-color: grey;
+					}
```

Comment:
there's a lot of extra spacing in the index.less file which makes it harder to read. Try formatting document before pushing the changes up.

Reply:
Beautified!